README.md

```diff
@@ -100,7 +107,37 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_accessor :name
+
+  def initialize name
+    @name = name
+  end
+
+  def greet
+    "Hello, I'm #{@name}"
+  end
+end
+
+pumba = Animal.new("Pumba")
+
+class Lion < Animal
+  attr_accessor :pack, :name
+
+  @@pack = 0
+
+  def initialize name, king
+    super(name)
+    @@pack += 1
+    @king = king
```

Comment:
Will give this to you, but we don't want to assume that a new lion with the name of "Simba" will also be given a king value of `true`. Let's use a conditional in `initialize` to make sure that "Simba" is kinged, like so...

```rb
def initialize name
  @name = name
  if @name == "Simba"
    @king = true
  end
end
```